lib/axlsx_styler.rb

```diff
@@ -8,3 +8,24 @@
 Axlsx::Workbook.send :include, AxlsxStyler::Axlsx::Workbook
 Axlsx::Worksheet.send :include, AxlsxStyler::Axlsx::Worksheet
 Axlsx::Cell.send :include, AxlsxStyler::Axlsx::Cell
+
+module Axlsx
```

Comment:
I would put this patch in a separate file (maybe axlsxs_styler/axlsx_package_patch.rb) and require it to maintain consistency with the way you wrote the rest of the gem. Or you could leave it here to help the code explicit.